test/scalability/README.md

```diff
@@ -66,3 +66,17 @@ make test-scalability
 ```
 
 Runs the scalability with minimalkueue and checks the results against `$(PROJECT_DIR)/test/scalability/default_rangespec.yaml`
+
+## Scrape result
+
+The scrape result `metricsDump.tgz` contains a set of `<ts>.prometheus` files, where `ts` is the millisecond representation of the epoch time at the moment each scrape was stared and can be used during the import in a visualization tool.
+
+If an instance of [VictoriaMetrics](https://docs.victoriametrics.com/) listening at `http://localhost:8428` is used, a metrics dump can be imported like:
+
+```bash
```

Comment:
It could be helpful to move it to a separate script, and from README just reference running the script.

Reply:
It's just an usage example with an almost random tool.

Reply:
We are likely to use it for checking performance with and without fair sharing. 

In the future we may use it for asserting the scalability in CI after the test, based on PromQL queries.

Reply:
In that future scenario it could be added, probably,  but for now it's not necessary.
(we are talking about 4 lines, two of which are more or less optional)

Reply:
ok, we can move it as future work, and for manual experimentation it is easy to copy-paste if needed.